src/plone/app/theming/transform.py

```diff
@@ -120,7 +121,9 @@ def parseTree(self, result):
             return None
 
         try:
-            return getHTMLSerializer(result, pretty_print=False)
+            return getHTMLSerializer(
```

Comment:
it's beyond the scope of this fix, but I was doing a transform yesterday and changed `getHTMLSerializer` from `repoze.xmliter` with `lxml.etree.HTMLParser`. Should we actually move to that one?

`repoze.itertools` does not look much taken care of 😓 

And actually it is just a tiny wrapper to `lxml`...

Reply:
Looks a nice idea to get rid of a dependency.
I created #240. I added the milestone 6.1 there, I would avoid doing that change for Plone 6.0 right now.